### ISSUE TITLE
Move /opportunity pages to /initiatives

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#2971]
At some point we decided to no longer use /opportunity and use /initiatives instead. Now we have projects arbitrarily split between both which is more confusing and complicated for staff and users.

Next steps:
- [x] confirm that we want to sunset /opportunity
- [x] move all /opportunity pages to /initiative
- [x] redirects those pages (one PR)
- [x] archive /opportunity (follow-up PR after the redirect PR lands)

**Comment by kristinashu** [comment]
Looping @anilkanji in on this for approval... 

I think this is something Jesse started a while ago. Originally, most staff would create pages under /campaign or /opportunity. But then a decision was made to use /initiatives instead of /opportunity (sorry, I don't have anymore insight into this but assuming it's because Initiatives sounds more accurate and is broader than Opportunity). The intention was to then sunset /opportunity but it just never happened, but only because of neglect and lack of prioritization. So now things are spread across both /opportunity and /initiatives and it's confusing to know which pages should go where. Sooo I think the time has come to move things over and sunset /opportunity. 

You ok with that? @alanmoo do you have any more context to add? Would you be able to socialize this with staff once it's happened?

**Comment by anilkanji** [comment]
To confirm - we're moving over /opportunity pages over to /initiatives?

Sounds good. Alan flagged the need for a broader IA review.

**Comment by kristinashu** [comment]
RIP ~/opportunity~

Long live /initiatives

**Comment by alanmoo** [comment]
I'm going to suggest a manually coded URL redirect rather than doing this in the Wagtail dashboard since there are so many pages. Let's set up the appropriate urls.py file to redirect anything at `/opportunity/*/*` to `/initiative/*/*` and then move the pages in the admin shortly after we deploy. (I think a few moments of downtime on those pages is ok)

**Comment by xmatthewx** [comment]
Good plan!

**Comment by mmmavis** [comment]
@kristinashu is this considered as part of the bigger IA work? If so, it's better that we tackle it along with other IA dev tickets (when they are ready for implementation).

**Comment by kristinashu** [comment]
@mmmavis this is something we're wanted to do for a very long time and I think the longer we leave it the more confusing and more work it will be to undo it later. So I don't think it should be blocked by the other IA work. Do you have any concerns about it?

**Comment by mmmavis** [comment]
@Pomax do we want to rename the model name to something other than `OpportunityPage`? If so, should it be done in a follow-up PR after we move the pages & land the redirect code?

**Comment by mmmavis** [comment]
@kristinashu can you clarify what you meant by "archive /opportunity"?

**Comment by kristinashu** [comment]
I mean that users should no longer be able to create pages under "opportunity" so it shouldn't even show up as an option to a users in the CMS.

**Comment by mmmavis** [comment]
PR for redirecting pages under `/opportunity` to `/initiatives` has passed code review. We will need to 

1. Go to production CMS and manually move those pages to `/initiatives`
2. Then merge and push the PR so old urls get redirected.

@kristinashu do you know when will be a good time to do this?

**Comment by kristinashu** [comment]
@mmmavis I can probably move the pages over next week. What to assign the ticket to me until I've done this?

**Comment by mmmavis** [comment]
@kristinashu done assigning! Right before you start moving the pages next week, can you let the engineering team know so we can be ready to merge and push the redirect PR live?

**Comment by kristinashu** [comment]
I've reviewed all the pages and I think they should all be moved directly to /initiatives.

@Pomax @mmmavis are you able to non-manually move all the pages and then redirect the old urls?

**Comment by mmmavis** [comment]
I can help with the manually move but I'm not comfortable running backend/database commands myself. Is there any concern with moving pages manually?

**Comment by kristinashu** [comment]
Just that there are a lot of pages 😭, especially considering the sub- pages and translations. And I thought Pomax said there was a way to do it, but maybe I misunderstood.

**Comment by xmatthewx** [comment]
@cadecairos - we have 18 wagtail pages and all their subpages to move and set up redirects. We're hoping to _not_ do it manually. Is this something you or Lucie might be able to help do? I wonder if Wagtail has a tool for bulk page management - I've seen that in other cms.

**Comment by cadecairos** [comment]
Wagtail has a management command that can be run to move pages: http://docs.wagtail.io/en/v2.0/reference/management_commands.html#move-pages

@Pomax can you coordinate running this command?

**Comment by mmmavis** [comment]
Yay! 🎉 @Pomax just helped moving all pages under `/opportunity` to `/initiatives`! Everything is working fine on prod now. 

I'll file a PR for the cleanup work (e.g., update fake data script etc)

**Comment by kristinashu** [comment]
w00t! Thank you so much for save me hours and hours of work! Also thank you for helping clean up the CMS and making it easier for all staff to use.

**Comment by mmmavis** [comment]
Should we remove the `opportunity` mini site name space from production CMS?

**Comment by kristinashu** [comment]
Yes! Please remove it.

**Comment by mmmavis** [comment]
Done! Thanks all! 🎉

**Comment by xmatthewx** [comment]
awesome! 🥇  thanks all for making that happen!
